styleguide/derek/pattern-components/cust-story-river/cust-story-river.ejs

```diff
@@ -0,0 +1,29 @@
+<div class="rsHomepage-customerStories-wrapper">
+    <div>
```

Comment:
Since we're not assigning a class or anything here I don't believe we need this `<div>` here.